index.html

```diff
@@ -115,6 +115,39 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+
+<!-- ________ Mocktic Joshi's Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Mocktic Joshi</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/mocktic" target="_blank">mocktic</a>
+          </p>
+          <p class="about">I love to code and build something fot the community</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://developer.mozilla.org/en-US/" target="_blank" title="First Resource">
+                  MDN Web Docs
+                </a>
+              </li>
+              <li>
+                <a href="https://stackoverflow.com/" target="_blank" title="Second Resource">
+                  Stackoverflow
+                </a>
+              </li>
+              <li>
+                <a href="https://www.w3schools.com/" target="_blank" title="Third resource">
+                  W3schools
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+<!-- ________ Mocktic Joshi's Contributor card ENDS ________  -->
```

Comment:
Place this comment line in line with your closing `<div>` tag, and make sure that there is only one line space between this comment line and the comment line of the next contributor's card. Suggest running `npm run prettier-html` on your code to format it correctly. Let us know if you have any questions.